bouncer/src/repo/repo_controller_internal.cpp.inl

```diff
@@ -38,6 +39,30 @@ RepoController::_RepoControllerImpl::_RepoControllerImpl(
 	{
 		subscribeToLogger(listeners);
 	}
+
+	// set logging directory
+	char* logDir = getenv("REPO_LOG_DIR");
+	std::string logPath;
+	logPath = logDir ? std::string(logDir) : "./log/";
```

Comment:
I don't think it's necessary to wrap a char* with the constructor, it should auto do it (I'm 100% not sure, give it a go)
```suggestion
	logPath = logDir ? logDir : "./log/";
```